hil/ext/switches/dellnos9.py

```diff
@@ -0,0 +1,380 @@
+# Copyright 2017 Massachusetts Open Cloud Contributors
```

Comment:
Do you need to consider renaming the hil.ext.switches.dell module to something more explicit now?